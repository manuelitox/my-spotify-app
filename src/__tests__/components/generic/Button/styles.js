import React from 'react'
import { shallow } from 'enzyme'

import StyledButton from 'components/generic/Button/styles'

describe('Component | StyledButton', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledButton />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})