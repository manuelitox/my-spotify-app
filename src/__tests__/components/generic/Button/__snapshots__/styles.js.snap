// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledButton should render a styled component 1`] = `
<StyledComponent
  bg="#F7F7F7"
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  background-color: ",
          [Function],
          ";
  border: 0;
  border-radius: 1px;
  color: ",
          [Function],
          ";
  display: block;
  font-size: 1.4rem;
  font-weight: 500;
  padding: 14px 20px;
  transform: skew(-4deg);
",
        ],
      },
      "defaultProps": Object {
        "bg": "#F7F7F7",
        "textColor": "#262D45",
      },
      "displayName": "styled.button",
      "foldedComponentIds": Array [],
      "propTypes": Object {
        "bg": [Function],
        "textColor": [Function],
      },
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "button",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  textColor="#262D45"
/>
`;
